syncnotification.go

```diff
@@ -108,7 +108,8 @@ func (mw *MultiWallet) fetchHeadersProgress(lastFetchedHeaderHeight int32, lastF
 
 	headersLeftToFetch := mw.estimateBlockHeadersCountAfter(lastFetchedHeaderTime)
 	totalHeadersToFetch := lastFetchedHeaderHeight + headersLeftToFetch
-	headersFetchProgress := float64(lastFetchedHeaderHeight) / float64(totalHeadersToFetch)
+	headersFetchedSoFar := mw.syncData.beginFetchTimeStamp
```

Comment:
How's the total number of headers fetched so far equal to the begin fetch timestamp?

Reply:
It does not, file has some changes.